src/Pages/Inventory/InventoryPage.css

```diff
@@ -0,0 +1,127 @@
+.search-bar-container {
+  margin: auto 5%;
+  text-align: center;
+  background: #fdfdfd;
+  border: 1px dashed rgb(196, 196, 196);
+  padding: 30px;
+  margin-bottom: 10px;
+}
+
+.table-container {
+  margin: auto 5%;
+  background: #fdfdfd;
+  border: 1px dashed rgb(196, 196, 196);
+  padding: 30px;
+  padding-bottom: 10px;
+}
+
+.spacer {
+  width: 100%;
+  padding: 10px;
+}
+
+.inventory-search-bar {
+  width: 80%;
+  margin: 0px;
+  padding: 5px 10px;
+  border-radius: 2px;
+  border: 1.5px solid grey;
+  box-sizing: border-box;
+  initial-letter-align: left;
+  vertical-align: middle;
+}
+
+.content-table {
+  border-collapse: collapse;
+  font-size: 0.9em;
+  max-width: 700px;
+  min-width: 700px;
+}
+
+.item-image {
+  max-width: 90px;
+  height: auto;
+  padding: auto;
+}
+
+.inventory-table {
+  table-layout: fixed;
+  width: 100%;
+}
+
+.inventory-table td {
+  word-wrap: break-word;
+}
+
+.inventory-table tr {
+  background-color: white;
+  color: black;
+}
+
+.inventory-table tr:hover {
+  background-color: #ebebeb;
+  color: black;
+}
+
+.inventory-row {
+  height: 100px;
+}
+
+.table-contents {
+  width: 100%;
+  height: 100px;
+  overflow: scroll;
```

Comment:
This is causing issues on Firefox. Is it possible to truncate instead?

<img width="1214" alt="Screen Shot 2021-05-08 at 2 24 07 PM" src="https://user-images.githubusercontent.com/61573792/117549618-6cb61a80-b009-11eb-9414-6b5aaf3725df.png">

Reply:
[There are options to hide scrollbars](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp), but I don't think we really want scrolling at all.